lib/lob/util.ex

```diff
@@ -46,6 +46,14 @@ defmodule Lob.Util do
   end
 
   @spec transform_argument({any, any}) :: list
+  defp transform_argument({:merge_variables, v}) do
+    [{"merge_variables", Poison.encode!(v)}]
+  end
+
+  defp transform_argument({"merge_variables", v}) do
```

Comment:
Can you change one of the tests to handle strings just for the little extra coverage? 

Maybe even change this function to be the following.

```ex
defp transform_argument({"merge_variables", v}), do: transform_argument({:merge_variables, v})
```